chapter_linear-networks/linear-regression-scratch.md

```diff
@@ -278,15 +278,13 @@ def sgd(params, grads, lr, batch_size):  #@save
 ```{.python .input}
 #@tab paddle
 #@save
-def sgd(params, lr, batch_size):
+def sgd(params, lr, batch_size):  
     """小批量随机梯度下降"""
-    revise_params=[]
     with paddle.no_grad():
-        for param in params:
-            param -= lr * param.grad / batch_size
-            param.stop_gradient = False
-            revise_params.append(param)
-        return revise_params
+        for i,param in enumerate(params):
+            param -= lr * params[i].grad/ batch_size
```

Comment:
Please follow the style guide.
`param -= lr * params[i].grad / batch_size`

Reply:
Done